src/main.rs

```diff
@@ -141,6 +158,12 @@ fn main() {
             if let Err(handled_pr) = handle_pull_request(pr, &bitbucket, &config.teamcity, &fanout) {
                 println!("{}{}", prefix(2), handled_pr);
             }
+
+            match &schedule {
+                &Some(ref sch) => sleep_duration = (sch.next_utc().unwrap() - UTC::now()).to_std().unwrap(),
+                _ => continue
+            }
```

Comment:
No worries. I've modified the `match` to bind a value to `sleep_duration` based on the existence of a `schedule` or not.